packages/core/src/connectors/index.ts

```diff
@@ -18,7 +18,7 @@ import { getConnectorConfig, readUrl, validateConnectorModule } from './utilitie
 // eslint-disable-next-line @silverhand/fp/no-let
 let cachedConnectors: LoadConnector[] | undefined;
 
-const loadConnectors = async () => {
+export const loadConnectors = async () => {
```

Comment:
If this function will be used elsewhere, better have a new name align with "virtual connector".